### PR TITLE
Added agrima

### PR DESCRIPTION
**Pull request by dhananjay425**

### CODE REVIEW

File: data/agrima.mdx [hunk 1]
```diff
@@ -0,0 +1,18 @@
+---
+name: Agrima
+description:The Company is working on Artificial Intelligence for the last couple of years and had developed a deep learning computer vision technology that could be used for building advanced image recognition solutions. We have used this technology in one of our Apps called Recipe Book. 
+founded: July 2012
+based: Both
+domain: IT Services and IT Consulting
+logo: agrima.png
```
Comment (lemokami) [inline data/agrima.mdx:7]: @dhananjay425 The logo filename is wrong

**Review by lemokami — CHANGES_REQUESTED** [review]

File: data/agrima.mdx (hunk 1 above)
Comment (dhananjay425) [reply data/agrima.mdx:7]: ohh i will correct that

File: data/agrima.mdx [hunk 2]
```diff
@@ -0,0 +1,18 @@
+---
+name: Agrima
+description:The Company is working on Artificial Intelligence for the last couple of years and had developed a deep learning computer vision technology that could be used for building advanced image recognition solutions. We have used this technology in one of our Apps called Recipe Book.
```
[the remark this answers is not in the document]
Comment (dhananjay425) [reply data/agrima.mdx:3]: sure

File: data/agrima.mdx (hunk 1 above)
Comment (dhananjay425) [reply data/agrima.mdx]: fixed

**Review by lemokami — APPROVED** [review]

**Comment by lemokami** [comment]
Thanks for the contribution @dhananjay425. A ⭐️ to the repo will be much appreciated. PR is now merged❤️

**Pull request merged**